src/components/RoomView/Clue.tsx

```diff
@@ -6,19 +6,23 @@ import { ref, child, push, update } from 'firebase/database';
 import { useSelector } from 'react-redux';
 import pluralize from 'pluralize';
 import { Button } from '@mui/material';
-
-const Clue = () => {
-  const [clueString, setClueString] = useState('');
-  const [clueNumber, setClueNumber] = useState(null);
-  const playerId = useSelector((state) => state.player.playerId);
-  const roomId = useSelector((state) => state.player.roomId);
-  const gameStatus = useSelector((state) => state.game.status);
-  const { teamOneSpymaster } = useSelector((state) => state.teamOne);
-  const { teamTwoSpymaster } = useSelector((state) => state.teamTwo);
-  const gameboard = useSelector((state) => state.wordsInGame.wordsInGame);
+interface ClueType {
+  clueString: string;
+  clueNumber: number;
+  }
+  
+const Clue:React.FC<ClueType> = () => {
+  const [clueString, setClueString] = useState<string>('');
+  const [clueNumber, setClueNumber] = useState<number | null>(null);
+  const playerId = useSelector((state:any) => state.player.playerId);
```

Comment:
just noting here that we will be able to change this now that redux is ts'd!
we can do this on seperate ticket